### PR TITLE
Testing out a multipage build based on experimental work

### PR DESCRIPTION
**Pull request by bryanjswift**
After discussion in schell/mogwai#43 I wanted to see how far I could get with an implementation based of the multiple route app.

Refs schell/mogwai#43

### CODE REVIEW

File: mogwai/src/view/dom.rs [hunk 1]
```diff
@@ -139,7 +140,8 @@ impl Drop for ViewInternals {
             }
             // TODO: Investigate whether we can drop rx responders before the element Rc
             // count drops to 1
-            self.rxs.iter().for_each(|rx| rx.drop_responder());
+            // TODO: Do responders need to explicitly be dropped?
+            // self.rxs.iter().for_each(|rx| rx.drop_responder());
```
Comment (bryanjswift) [inline mogwai/src/view/dom.rs]: This made the example work but I don't know what bad things this does if responders are not explicitly dropped.

Comment (schell) [reply mogwai/src/view/dom.rs]: Interesting! So you had to comment this out for the example to work? In that case - this will make a nice unit test.

**Comment by schell** [comment]
If you make this PR into my clone (instead of your own) then I'll accept it and distill it into a unit test.

**Comment by bryanjswift** [comment]
> If you make this PR into my clone (instead of your own) then I'll accept it and distill it into a unit test.

@schell will do! I didn't want to pollute the space since I knew this was a work in progress branch.

**Comment by bryanjswift** [comment]
Closing to reopen against the upstream

**Pull request closed without merging**